Core/src/org/sleuthkit/autopsy/modules/sevenzip/SevenZipIngestModule.java

```diff
@@ -657,24 +661,7 @@ private boolean isSupported(AbstractFile file) {
      * @return true if zip file, false otherwise
      */
     private boolean isZipFileHeader(AbstractFile file) {
-        if (file.getSize() < readHeaderSize) {
-            return false;
-        }
-
```

Comment:
maybe this is what tika does under the hood, but this method name is misleading now...